implementations/rust/ockam/ockam/src/profile/channel.rs

```diff
@@ -2,6 +2,8 @@ use ockam_core::{Address, Result, Route};
 use ockam_node::Context;
 
 mod listener;
+use crate::profile::ProfileAuth;
+use crate::profile::ProfileContacts;
```

Comment:
How should we handle trait explosion like this?

- Per crate preludes? `use crate::profile::prelude::*"?
- Just accept it and import what's necessary?
- ?
-

Reply:
Internally, I think we should import what's necessary and keep dependencies explicit.
We can explore prelude::* for user facing api incase it becomes necessary ... but I suggest we avoid it as long as we can.